BlueprintUI/Sources/Extensions/BidirectionalCollection+Blueprint.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+extension BidirectionalCollection where Element: Equatable, Element: NSObjectProtocol {
```

Comment:
Nit: I think you can also combine the constrains
```suggestion
extension BidirectionalCollection where Element: Equatable & NSObjectProtocol {
```

Reply:
Ah, of course!
 I  initially wrote `Element: Equatable, NSObjectProtocol ` and this was Xcode's suggested fix.